### ISSUE TITLE
Improve our guidance on creating new journals/publications

### ISSUE THREAD

**Issue by arfon** [openjournals/governance#3]
Currently we have [very limited specifications/requirements](https://github.com/openjournals/governance/blob/master/governance.md#editorial-policies-and-processes) on what Open Journals publications/journals should be like:

> - Be open access
> - Have an open review process
> - Use the Open Journals open source toolchain

It would be good to expand on these guidelines and potentially include information on:

- **Branding:** Should Open Journals publications follow a particular branding guideline?
- **Funding:** Running a journal isn't free. DOIs cost money, web servers cost money etc. Should we say something about our expectations about covering these costs?
- **Code of Conduct:** As a NumFOCUS project we are subject to a CoC. This means Open Journals publications are subject to this too - we need to be sure they have a clear statement about conduct and mechanisms in place for handling potential violations within their respective communities.

**Comment by danielskatz** [comment]
Re funding, I would like to see something tiered potentially, where small efforts don't have to pay, but perhaps larger ones do

**Comment by labarba** [comment]
> I would like to see something tiered potentially, where small efforts don't have to pay

Yes, but _we_ have to pay. This is putting the onus on TheOJ to "sponsor" the small efforts that we welcome to use our infra, which makes it more pressing that we be selective.

**Comment by labarba** [comment]
In general, we need to articulate a process for considering new journals, and perhaps develop an application form. 

What about journal topics? Is TheOJ going to focus on scientific journals? Or will it be open to any subject (philosophy and what not). Targeted scope is probably better. 

We need to require that they publish ethical guidelines, like we developed for JOSS. We may hint at diversity of the editorial board, and require a minimum number of members (at least five?). We might ask that they be willing to contribute to the general health and maintenance of TheOJ project, somehow.

**Comment by danielskatz** [comment]
> What about journal topics? Is TheOJ going to focus on scientific journals? Or will it be open to any subject (philosophy and what not).

I don't think there's anything specific we are doing with regards to science - we accept software in non-science fields now in JOSS.  Why would we limit this?

**Comment by kyleniemeyer** [comment]
I don't think we need to limit potential journals to STEM fields. After all, JOSS doesn't focus on "science" software, but more broadly "research" software which can come from essentially any field.
